lib/puppet/provider/exec/powershell.rb

```diff
@@ -1,5 +1,5 @@
 require 'puppet/provider/exec'
-require_relative '../../../puppet_x/puppetlabs/powershell_manager'
+require 'puppet_x/puppetlabs/powershell_manager'
```

Comment:
This will work for agents (because they add the pluginsync'ed lib directory to $LOAD_PATH). But it will cause catalog compilations to fail on the master. The master will `require` the powershell provider, but will be unable to require the `powershell_manager` helper code. This is why we prior to `require_relative` we had two different code paths for loading helper code, e.g. https://github.com/puppetlabs/puppetlabs-registry/blob/master/lib/puppet/provider/registry_key/registry.rb#L8-L11

Note if the agent on the master has done a pluginsync, then you won't see the problem. But if you delete `puppet agent --configprint libdir` on the master you will.